packages/react-admin-core/src/table/useTableQueryPaging.tsx

```diff
@@ -5,7 +5,7 @@ export interface IPagingApi<T> {
     init: T;
     current: T;
     currentPage?: number;
-    changePage: (variables: T, page?: number) => void;
+    changePage: (variables: T, page?: number, noScrollToTop?: boolean) => void;
     attachTableRef: (ref: React.RefObject<HTMLDivElement | undefined>) => void;
```

Comment:
Nach einem 'page-change' sollte man die Filter jedenfalls sehen.

Ich seh da 2.Möglichkeiten:

1. bei 'page-change' zu top-of-page scrollen und nicht zu tableRef  (`window.scrollTo(0,0)`)
2. user kann das dom-element zu dem gescrollt wird selbst definieren, das kann dan ein table-wrapper sein indem die Filter inkludiert sind, dazu könnte man attachTableRef zu attachScrollToRef ändern




```suggestion
    attachScrollToRef: (ref: React.RefObject<HTMLDivElement | undefined>) => void;
```

Reply:
Meine Meinung:

Warum soll er die Filter sehen wollen? Die haben sich ja nicht geändert. So wie es jetzt ist hat er maximal viel platz für die tabellendaten im viewport.

zu den 2 Möglichkeiten:

1. scrollTo(0,0) ist in manchen fällen nicht möglich (wenn mehrere tables untereinander sind, zB dashboards - mobidrome hatten wir das glaub ich schon)

2. wenn das optional ist und das standard verhalten so bleibt können wir das gerne ändern

Reply:
Möglichkeit 1 find ich nicht gut, weil sich Filter + Tabelle nicht zwingend ganz oben auf der Seite befinden müssen

Möglichkeit 2 gefällt mir besser, das hab ich auch überlegt. Aktuell ist hier das Problem, dass sich die Table jedes Mal, wenn sie gerendered wird selbst attached (https://github.com/vivid-planet/react-admin/blob/master/packages/react-admin-core/src/table/Table.tsx#L151).
Aber ich kann sonst mal probieren, das überschreibbar zu machen